pkg/get/tools.go

```diff
@@ -456,13 +456,13 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}-{{.Version}}-{{$osStr}}-{{$archStr}}.{{$extStr}}`,
 		})
 
-	// https://releases.hashicorp.com/terraform/0.13.1/terraform_0.13.1_linux_amd64.zip
+	// https://releases.hashicorp.com/terraform/0.14.3/terraform_0.14.3_linux_amd64.zip
```

Comment:
@jsiebens / @Waterdrips does this change break any of our faasd tutorials?

Reply:
The tf 0.14 upgrade docs don't mention any breaking changes to config syntax. Might be worth checking them with 0.14 though.